datadog_checks_downloader/tests/scripts/download_test_data.py

```diff
@@ -63,19 +64,19 @@ def filenames_for_target(metadata_file, target):
 
             target_data = metadata['signed']['targets'][target]
 
-            yield hashed_target_files[target]
+            yield from hashed_target_files[target]
             for name in target_data['custom'].get('in-toto', []):
-                yield hashed_target_files[name]
```

Comment:
As a python noob, I'm a bit confused why the change from `yield` to `yield from`? From what I understand, `yield from bar` is similar to `for foo in bar: yield foo`, although nothing else in this function changed?

Reply:
I'm also not sure that `hashed_target_files[target]` and `hashed_target_files[name]` are iterables?

Reply:
This function hasn't changed, but the data it operates on has. `hashed_target_files[target]` is now a list, holding all possible paths (with different sha's) for that target. https://github.com/DataDog/integrations-core/blob/81919ecbb1425f9343986a2ad7f724b86f02c8c2/datadog_checks_downloader/tests/scripts/download_test_data.py#L75-L79